### DISCUSSION TITLE
I was trying to update codespaces using credential manager..

### DISCUSSION THREAD

**Discussion by garthlearnspython** [community/community#38602] [Codespaces]
I used the credentials manager to change my password to the new key that i was prompted to switch to,  it was not working. I kept getting a prompt to re login and when using my password it didn't work. I tried using the key as my password and it did not work, i ended up looking at forums that suggested switching my password to the key using credential manager and now i cannot reconnect to do  my homework in codespaces PLEASE help. This is using visual studio code trying to connect cs50 codespaces

**Comment by dmilburn** [comment]
Hi from the codespaces team @garthlearnspython.

A few ideas:
- Would it work for you to use codespaces in the web instead of in VSCode? If you go to github.com/codespaces you should see your codespace on that page. If you click the "..." on the right side of that codespace line item, you should be able to click "open in" and then "Open in Browser"
- If you're completely locked out of your GitHub account, consider resetting your password: https://github.com/password_reset 
- It may be worth also checking if your VSCode and Codespaces Extension versions are both up to date

Let me know if any of those help!

**Comment by garthlearnspython** [comment]
Hi, thanks for your response i did get it working on the codespaces through
my browser and i was able to properly verify the new way of logging in, i
still have to check if this fixed my issue with using visual studio code
